### PR TITLE
chore(pml): Remove filter toggle

### PR DESCRIPTION
**Pull request by jaredvu**
Launchable markets should always be available and not hidden on `MarketsTable` and `MarketsDropdown`

---

## Views

- `<MarketFilter>`
  - remove slotBefore component that displays `<Switch>` to hide launchable markets

## Hooks

- `hooks/useMarketsData`
  - remove filter logic that uses redux state for `shouldHideLaunchableMarkets`

## State
- `state/appUiConfigs`
  - Remove  `shouldHideLaunchableMarkets` and action

## Migration

- `4.ts` and `4.test.ts`
  - Add migration to remove `shouldHideLaunchableMarkets` state

---

### CODE REVIEW

File: src/state/migrations.ts [hunk 1]
```diff
@@ -5,12 +5,15 @@ import { migration0 } from './migrations/0';
...
 import { migration2 } from './migrations/2';
 import { migration3 } from './migrations/3';
+import { migration4 } from './migrations/4';
 
 export const migrations: MigrationManifest = {
   0: migration0,
   1: migration1,
   2: migration2,
   3: migration3,
+  // @ts-expect-error PersistAppStateV4 is a union of PersistedState and new app state
```
Comment (tyleroooo) [inline src/state/migrations.ts:15]: why is the compiler angry?

Comment (jaredvu) [reply src/state/migrations.ts:15]: The fn expects PersistedState and not a union of `PersistedState & { appUiConfigs: ... }`

Comment (tyleroooo) [reply src/state/migrations.ts:15]: the other ones did the same thing, why is it failing now?

Comment (tyleroooo) [reply src/state/migrations.ts:15]: I think I'd prefer the function be the type this thing wants (take PersistedState | undefined as the argument) then inside the function cast PersistedState to PersistAppStateV3. That's generally safer than totally turning off the compiler checks for a line.

Comment (tyleroooo) [reply src/state/migrations.ts:15]: here's someone else doing a similar thing:
```
const manifest: MigrationManifest = {
    1: (state: PersistedState) => {
        const persistedState = state as Root<VERSION_0>;
        const migratedState: Root<VERSION_1> = migrateState1(persistedState);
        return migratedState;
    },
    2: (state: PersistedState) => {
        const persistedState = state as Root<VERSION_1>;
        const migratedState: Root<VERSION_2> = migrateState2(persistedState);
        return migratedState;
    },
    3: (state: PersistedState) => {
        const persistedState = state as Root<VERSION_2>;
        const migratedState: Root<VERSION_3> = migrateState3(persistedState);
        return migratedState;
    }
}
```

Comment (jaredvu) [reply src/state/migrations.ts]: Helper function added to cast param types

**Review by tyleroooo — APPROVED** [review]

**Pull request merged**